MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,1293 @@ This page lists the keys for each control and the element to which it is mapped
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+        <tr>
+        <td>
+            SfShimmerFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Fill color of the shimmer.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	</tr>
+    <tr>
+        <td>
+            SfShimmerWaveColor 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Wave color of the shimmer.
+        <br/>
+        <br/>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SfShimmerNormalBackground  
+            <br/>
+            <br/>
+        </td> 
+        <td>
+            Background color of the shimmer.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+</table>
+
+## SfLinearProgressBar
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="14">
+            SfLinearProgressBarStyles  
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfLinearProgressBarTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfLinearProgressBar without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfLinearProgressBarTheme">CommonTheme</x:String>
+                <Color x:Key="SfLinearProgressBarTrackColor">Pink</Color>
+                <Color x:Key="SfLinearProgressBarProgressColor">Purple</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+        <tr>
+        <td>
+            SfLinearProgressBarTrackColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Trach color of the linear progress bar.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	</tr>
+    <tr>
+        <td>
+            SfLinearProgressBarBackground 
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Progress bar background of the linear progress bar.
```

Comment:
**Progress bar** background of the **linear progress bar.**

Progress bar mentioned - continusouly